File: igraph/road-net-pa.py
import core

PATH = '../samples/roadNet-PA.txt'
AVOID_DOUBLES = True
UNIQUE_TESTS = [100, 1000, 10000, 100000]
INDEXES = [100, 1000, 10000, 100000]
ALGORITHMS = ['bfs', 'dfs', 'shortest']

core.run(PATH, AVOID_DOUBLES, UNIQUE_TESTS, INDEXES, ALGORITHMS)

File: cytoscape/road-net-pa.js
var core = require('./core');

const path = '../samples/roadNet-PA.txt';
const avoidDoubles = true;
const uniqueTests = [100, 1000, 10000, 100000];
const indexes = [100, 1000, 10000, 100000];
const algorithms = ['bfs', 'dfs', 'shortest'];

core.run(path, avoidDoubles, uniqueTests, indexes, algorithms);